src/platform/site-wide/va-footer/helpers.jsx

```diff
@@ -24,13 +24,25 @@ export function generateLinkItems(links, column, direction = 'asc') {
   const captureEvent = () => recordEvent({ event: FOOTER_EVENTS[column] });
   return (
     <ul className="va-footer-links">
-      {orderBy(links[column], 'order', direction).map(link => (
-        <li key={`${link.column}-${link.order}`}>
-          <a href={link.href} onClick={captureEvent} target={link.target}>
-            {link.title}
-          </a>
-        </li>
-      ))}
+      {orderBy(links[column], 'order', direction).map(link => {
+        // added this to deal with the extra label on the footer
+        // due to design change.
+        if (link.href) {
+          return (
+            <li key={`${link.column}-${link.order}`}>
+              <a href={link.href} onClick={captureEvent} target={link.target}>
+                {link.title}
+              </a>
+            </li>
+          );
+        }
+
+        return (
+          <li key={`${link.column}-${link.order}`}>
+            <h6>{link.title}</h6>
+          </li>
+        );
+      })}
```

Comment:
I'm not sure, but I think the label should technically be a part of the same list element as the link for accessibility. Maybe we could add an optional `label` property to the JSON?

@RobAWilkinson @jbalboni What do you think?

Reply:
Yeah, a list item that's just a header is confusing. Also, I don't think the TTY: 711 number should be linked, based on what Beth said. I think it's direction for if you use the phone number and you have a TTY phone.

I think we should go back in the direction of the Vets.gov markup. This really isn't a list of links anymore.

Reply:
Great, I will make the updates. I felt like this was kinda weird but I was trying to get it in yesterday. I'm glad we have a little more time.

Reply:
@jbalboni Regarding the TTY. Let me find out because the mockup shows it as being a link. Let me ask Beth.